### ISSUE TITLE
Project Creator: Display settings are not stored

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#441]
@K-Meech 

Is seems as if the display settings (min max) of the currently open image are not properly transferred to the MoBIE project:

<img width="748" alt="image" src="https://user-images.githubusercontent.com/2157566/133251322-f310c52d-d7c2-45ed-8774-9c3b96090c95.png">

**Comment by K-Meech** [comment]
Yes - currently it just sets to the defaults for the bitdepth i.e. 8-bit = 0 and 255 as the limits.
I'll look into keeping the current settings.

**Comment by tischi** [comment]
We should also fetch the color of the LUT.

**Comment by K-Meech** [comment]
I'm not sure if fetching the LUT is worth it. We only support a small number (and most of these aren't in base fiji, I think?). So it would only be glasbey that could be matched?

**Comment by tischi** [comment]
I don't mean multi-color LUTs at this point, I mainly mean just the color, e.g. `gray` or `blue` or the RGB values. I am pretty sure the `ImagePlus` has a getter method for something like this.

**Comment by K-Meech** [comment]
Ah ok - I'll take a look!
